plugins/modules/smbshare.py

```diff
@@ -773,8 +957,16 @@ def get_powerstore_smb_share_parameters():
         is_branch_cache_enabled=dict(type='bool'),
         is_continuous_availability_enabled=dict(type='bool'),
         is_encryption_enabled=dict(type='bool'),
-        state=dict(required=True, choices=['present', 'absent'],
-                   type='str'),
+        state=dict(required=True, choices=['present', 'absent'], type='str'),
+        acl=dict(
+            type='list', elements='dict',
+            options=dict(
+                state=dict(type='str', required=True, choices=['present', 'absent']),
```

Comment:
for state, can we give a default value as present?

Reply:
by default made it mandatory because all the fields are mandatory in acl